Dev-Corner/slides/toolshed.html

```diff
@@ -0,0 +1,861 @@
+                            - [Galaxy community Tool Shed](https://toolshed.g2.bx.psu.edu/) -> Main Tool Shed
+                            - [Galaxy community test Tool Shed](https://testtoolshed.g2.bx.psu.edu/) -> Sandbox for testing
+                            - [List of public Tool Sheds](https://wiki.galaxyproject.org/PublicGalaxyServers#Public_ToolSheds)
+                            - [IFB Tool Shed](http://toolshed.france-bioinformatique.fr/)
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Tool Shed interface
+                            <img src="../images/main_toolShed_ss.png" class="no-border">
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Example of tool
+                            <img src="../images/exampletool_ss.png" class="no-border" width="80%">
+
+                        </script>
+                    </section>
+                </section>
+                <section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ## 2. Installing tools from a Tool Shed
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Connect your Galaxy to a Tool Shed
+                            - Connect your Galaxy to a Tool Shed in `config/tool_sheds_conf.xml`:
+                            ```
+                              <tool_sheds>
+                                <tool_shed name="Galaxy main tool shed" url="https://toolshed.g2.bx.psu.edu/" />
+                              </tool_sheds>
+                            ```
+                        </script>
+                    </section>
+                    <section data-markdown>
+                        <script type="text/template">
+                            ### Connect your Galaxy to a Tool Shed
+                            - Set dependencies resolution
+                              - in `config/galaxy.ini`:
+                            ```
+                            conda_prefix = <tool_dependency_dir>/_conda/
+                            conda_auto_install = True
+                            conda_auto_init = False
+                            tool_dependency_dir = ../tool_dependencies
+                            ```
+                              - in config/dependency_resolvers_conf.xml (be careful order matters):
+                            ```
+                            <dependency_resolvers>
+                                  <conda />
+                                  <conda versionless="true" />
+                                  <tool_shed_packages />
```

Comment:
By experience, I have change my mind.
The order should remain inchanged.
You can choose if you want the tool_dependencies during the tool installation. If so, Galaxy will take it.
If not, Galaxy will search for a conda package.

If we change the order, Galaxy may find a conda package in the good version but unfortunatly, the wrapper is not ready for it and still need the dedicaded tool_dependency